app/Providers/AppServiceProvider.php

```diff
@@ -46,6 +53,12 @@ class AppServiceProvider extends ServiceProvider
 	 */
 	public function boot()
 	{
+		if (Env::get('DB_LOG_SQL', false)) {
```

Comment:
I would discourage the use of `ENV`
which I assume is the same as the shortcut `env()`
https://stackoverflow.com/questions/34263107/get-environment-value-in-controller/54286878#54286878

`config()` is preferable here.

You could add a `db_log_sql` in the `config/app.php`
which takes values `env('DB_LOG_SQL', false)`
and call here `config('app.db_log_sql')`.

The behaviour is then safer.

Reply:
Actually this is a left over. We may as well remove this code completely. I needed that code for debugging. However `config()` and `env` are not equivalent here and choosing `env` was intentionally. I needed to debug the SQL queries during the unit tests (i.e. `make test` or `php ./artisan test`) and PHPunit does not use the configuration but sets `DB_LOG_SQL` as an environment variable.

Reply:
ohhh interesting. Did not know that.

Reply:
So, what shall we do? Just remove the code?

Reply:
As I don't need it anymore, I rewrote it as you suggested.